### PR TITLE
Add cultural change to activism

### LINKED ISSUE #2475: Adding a chapter on cultural change to 'Ethical Research'
## Summary

We do not yet have information on how to achieve cultural change in a community/organisation. I recently followed a course on this and want to collate some resources on this so I was thinking of adding a chapter to the Ethical Research Guide where it seems to fit best. 

### Discussion Points

- [x] Is the Ethical Research Guide indeed the best place? If not, open to suggestions!
- [ ] ?

## Who can help?

* Anyone with experience achieving cultural change 
* Beginners with this subject to ensure accessibility/clarity of the chapter

## Updates

- [x] Write chapter outline (WIP: https://hackmd.io/@TycZoDDEQIq6yttGuqbuww/ryjfaOK3q)
- [x] Add material to the chapter
- [x] Combine materials into a readable chapter
- [ ] Proofread
- [ ] Request reviews
- [ ] Address reviews
- [ ] Merge to main branch.

### PR DESCRIPTION
**Pull request by EstherPlomp**
### Summary

Adding Cultural Change as a sub-section to Research Activism

Fixes #2475

### List of changes proposed in this PR (pull-request)

* Add cultural change to the Research Activism summary
* Add a new section on cultural change to the book.

### What should a reviewer concentrate their feedback on?

- [x] Is this the right place for a section on Cultural Change? 
- [ ] Does the text flow? 
- [ ] Do any resources miss?
- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by aleesteele** [comment]
@EstherPlomp, just wanted to Kirstie wrote a document before she went on leave about 'Emergence as The Turing Way's strategy', which is on a HackMD here: https://hackmd.io/@KirstieJane/rJ0jRCaEc

I really love that you are adding in this work on cultural change, as it speaks to how deeply embedded these ideas are within the community. Will add in more comments asap, but just wanted to note this first!

**Comment by EstherPlomp** [comment]
> @EstherPlomp, just wanted to Kirstie wrote a document before she went on leave about 'Emergence as The Turing Way's strategy', which is on a HackMD here: https://hackmd.io/@KirstieJane/rJ0jRCaEc
> 
> I really love that you are adding in this work on cultural change, as it speaks to how deeply embedded these ideas are within the community. Will add in more comments asap, but just wanted to note this first!

Thanks @aleesteele! Going over it now and adding some notes here so that I do not forget: 
- Brown uses Obolensky’s definition of emergence as “the way complex systems and patterns arise out of a multiplicity of relatively simple interactions”.

File: book/website/ethical-research/activism/activism-cultural-change.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+## What is cultural change?
+
+“inspiring a change in behaviour in persons/organisations, with a lasting impact, strengthening the core values of persons/organisation
+
+(er-activism-cultural-change-emergent-strategy)=
+
+## Emergent strategy
+
+Emergent strategy by marie adrienne brown teaches us that change is inevitable, and that we can shape change.
```
[the remark this answers is not in the document]
Comment (Arielle-Bennett) [reply book/website/ethical-research/activism/activism-cultural-change.md:15]: Also we should probably cite/link to the book here :)

File: book/website/ethical-research/activism/activism-cultural-change.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+*	What you pay attention to grows
+
+Conversation is a crucial way to explore what we believe and to make new understandings and ideas possible. 
+Consider the areas where trust needs to be built: you will need to demonstrate that you're thinking about the same values/concerns. 
+To accomplish your vision for the future it needs to be clear what needs to be done and who will do it. 
+There should be time for reflection and evaluation to apply lessons learned to future work, and time for celebration of collective achievements. 
+
+(er-activism-cultural-change-transitions)=
+## Managing Transitions
+
```
Comment (Arielle-Bennett) [inline book/website/ethical-research/activism/activism-cultural-change.md:39]: Also a note to include citation/link in final version

File: book/website/ethical-research/activism/activism-cultural-change.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+Managing Transitions by William Bridges teaches us that a change will always be accompanied by a transition. A transition begins when something needs to be let go, and sits in between the old and the new isutation. This transition phase can be insecure for people who will have to adjust to a new situation, but also an opportunity for innovation to happen if people are adequately resourced and supported (by policies, groupings, procedures).
```
[the remark this answers is not in the document]
Comment (Arielle-Bennett) [reply book/website/ethical-research/activism/activism-cultural-change.md:40]: Also sp: isutation - situation?

File: book/website/ethical-research/activism/activism-cultural-change.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+## Managing Transitions
+
+Managing Transitions by William Bridges teaches us that a change will always be accompanied by a transition. A transition begins when something needs to be let go, and sits in between the old and the new isutation. This transition phase can be insecure for people who will have to adjust to a new situation, but also an opportunity for innovation to happen if people are adequately resourced and supported (by policies, groupings, procedures). 
+
+To manage transitions it needs to be clear what the problem (reason for change) is. People will not change if they are not experiencing the problem and are convinced something needs to be adjusted. Those experiencing the problem will also have expertise and knowledge to come up with an effective solution. Individuals will stand to loose something when change is happening and a sense of control over this transition may help making transition phases less uncomfortable. It is important to learn what problems people are facing in a trancition face, and recognise individuals that do contribute to the change. People will need to feel safe to experiment and take risks, and not be punished when these experiments fail.
```
Comment (Arielle-Bennett) [inline book/website/ethical-research/activism/activism-cultural-change.md:42]: what problems people are facing in a **trancition** face - what problems people are facing in a transition face

File: book/website/ethical-research/activism/activism-cultural-change.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+It is important that changes are meaningfully clustered: avoid further changes to avoid a messy change pile. 
+
+Short term goals can give people a sense of achievement and encourage them to move forward.  
+
+To ensure that people will change their behaviour, they will need to learn how to adapt to new behaviours through training. People will not be able to invest in this, if the new behaviours are not adequately rewarded (for example, focused placed on publications written instead of open data/software).
+
+There are four P's in transitions (see Managing Transitions by William Bridges):
```
Comment (Arielle-Bennett) [inline book/website/ethical-research/activism/activism-cultural-change.md:50]: Think this $ Ps summary would be better up the top of this section

File: book/website/ethical-research/activism/activism-cultural-change.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
+(er-activism-cultural-change)=
+
+# Cultural Change
+
+(er-activism-cultural-change-what)=
+
+## What is cultural change?
+
+“inspiring a change in behaviour in persons/organisations, with a lasting impact, strengthening the core values of persons/organisation
```
Comment (Arielle-Bennett) [inline book/website/ethical-research/activism/activism-cultural-change.md:9]: might be useful up here to describe some situations for cultural change and where researchers might encounter them? Sort of expanding on this with a "why is culture change and associated concepts important for data scientists to understand?" paragraph

File: book/website/ethical-research/activism/activism-cultural-change.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+There are four P's in transitions (see Managing Transitions by William Bridges): 
+- Purpose: people have to understand the logic behind the outcome and feel that the change is necessary
+- Picture: people need to experience how the outcome will look and feel
+- Plan: people need a step-by-step plan to know what they have to do
+- Part to play: especially if people have something to loose, they need to be able to play a part in the transition phase and eventual outcome
+
+(er-activism-cultural-change-leading)=
+## Leading change
+
+According to John P Kotter there are eight steps in change:
```
Comment (Arielle-Bennett) [inline book/website/ethical-research/activism/activism-cultural-change.md:59]: Note on citing/linking (these are mostly so we don't forget when doing next draft)

**Review by Arielle-Bennett — COMMENTED** [review]
Love that this chapter is being written! I think it's a really important one for open scientists and researchers, but an area that is often overlooked. 

Good start here - have added some comments and happy to discuss other points that might make it into the chapter here as well. 

For example, I think having sections explaining the relevance of culture change for researchers is crucial, and also contextualising the framings (these are some ways of thinking about and approaching change which can be helpful to articulate a strategy)

**Comment by aleesteele** [comment]
> Can we actually link to this? I think this document on its own will be of help to others!

@EstherPlomp: Just adding that yes! We definitely plan to add this to the book, but the idea is that it would be in the forward chapter, which is a WIP – hence the delay in doing so!

File: book/website/ethical-research/activism/activism-cultural-change.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (EstherPlomp) [reply book/website/ethical-research/activism/activism-cultural-change.md:40]: Woops, sorry for the mess and thanks for spotting!

File: book/website/ethical-research/activism/activism-cultural-change.md [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+
+To manage transitions it needs to be clear what the problem (reason for change) is. People will not change if they are not experiencing the problem and are convinced something needs to be adjusted. Those experiencing the problem will also have expertise and knowledge to come up with an effective solution. Individuals will stand to loose something when change is happening and a sense of control over this transition may help making transition phases less uncomfortable. It is important to learn what problems people are facing in a trancition face, and recognise individuals that do contribute to the change. People will need to feel safe to experiment and take risks, and not be punished when these experiments fail.
+
+It is important that changes are meaningfully clustered: avoid further changes to avoid a messy change pile.
```
[the remark this answers is not in the document]
Comment (EstherPlomp) [reply book/website/ethical-research/activism/activism-cultural-change.md:44]: I tried to elaborate a bit more! Thanks :)

File: book/website/ethical-research/activism/activism-cultural-change.md (hunk 5 above)
Comment (EstherPlomp) [reply book/website/ethical-research/activism/activism-cultural-change.md:50]: Even before the introduction into transitions?

File: book/website/ethical-research/activism/activism-cultural-change.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (EstherPlomp) [reply book/website/ethical-research/activism/activism-cultural-change.md]: Yes indeed - updated the bibliography now and added references. Thanks!

File: book/website/ethical-research/activism/activism-cultural-change.md (hunk 6 above)
Comment (EstherPlomp) [reply book/website/ethical-research/activism/activism-cultural-change.md]: Working on this - should probably improve but I need to move on for today :( Thanks!

**Comment by EstherPlomp** [comment]
> > Can we actually link to this? I think this document on its own will be of help to others!
> 
> @EstherPlomp: Just adding that yes! We definitely plan to add this to the book, but the idea is that it would be in the forward chapter, which is a WIP – hence the delay in doing so!

So then it is probably better to wait until we can refer to the foreword in this section, right? :)

**Comment by EstherPlomp** [comment]
> Love that this chapter is being written! I think it's a really important one for open scientists and researchers, but an area that is often overlooked.
> 
> Good start here - have added some comments and happy to discuss other points that might make it into the chapter here as well.
> 
> For example, I think having sections explaining the relevance of culture change for researchers is crucial, and also contextualising the framings (these are some ways of thinking about and approaching change which can be helpful to articulate a strategy)

Thanks for reviewing @Arielle-Bennett ! I still need to work now on the how does this matter to you as a data scientist and I have hopefully addressed most of your other comments. Happy week off for now :)

File: book/website/ethical-research/activism/activism-cultural-change.md [hunk 9]
```diff
@@ -0,0 +1,127 @@
...
+When you're affected by cultural change taking place in your environment, it may be difficult to change your practises and behaviour. 
+Cassandra Worthy wrote 'Change Enthusiasm' ({cite:ps}`Worthy2021change`) to provide you with exercises and tools to see changes as an opportunity for learning and growth.
+As an Early Career Researcher you may also have to deal with systemic cultural change in your discipline. 
+The work by {cite:ps}`Kent2022change` contains recommendations that may support you in this change journey.
+You might be involved at your institute in a committee that is trying to update a policy, or add a new one (see the [Guide to Accelerate Public Access to Research Data](https://www.aplu.org/news-and-media/News/aplu-and-aau-issue-guide-to-accelerate-public-access-to-research-data) for some pointers that are applicable beyond a research data policy). 
+You may be advocating to make your work environment more inclusive, at the office or within your research association.
+
+Below follow short summaries of work on cultural change, written by adrienne marie brown, William and Susan Bridges and John P Kotter.
```
Comment (malvikasharan) [inline book/website/ethical-research/activism/activism-cultural-change.md:27]:
```suggestion
Below follow short summaries of work on cultural change, written by Adrienne Marie Brown, William and Susan Bridges and John P. Kotter. 
```

**Comment by EstherPlomp** [comment]
Closing this PR since I didn't manage to fix the Xs - but it works here: #2875

**Pull request closed without merging**